public/javascript/posts/BookAppointment.js

```diff
@@ -19,6 +19,24 @@ var app = angular.module('BookAppointmentApp');
     ];
 
     $scope.viewRenderWeekAgenda = function(view, element){
+      if(view.type == 'month'){

```

Comment:
dayEvent is called when we actually click on a day after the month view is loaded. alertOnEventClick is called when we click on the existing appointment after the month view is loaded.